frontend/src/components/creator-module/creator-edit/creator-edit.tsx

```diff
@@ -60,6 +63,7 @@ const CreatorEdit = ({ newQuestion, setNewQuestion }: Prop): ReactElement => {
             className="question-title-input"
             placeholder="eg: Question 1 Title"
             defaultValue={previewQuestion.title}
+            value={newQuestion.title}
```

Comment:
add `maxLength` attribute to the input element to limit characters.
for example `maxlength={50}`